.github/workflows/push.yaml

```diff
@@ -24,8 +24,19 @@ jobs:
           password: ${{ secrets.DOCKER_PASSWORD }}
       - name: Build and push
         uses: docker/build-push-action@v3
+        id: build-and-push
         with:
           context: .
           platforms: linux/amd64,linux/arm64
           push: true
           tags: webispy/checkpatch
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.repository.default_branch }}
+      - name: Update Dockerfile in default branch with digest
+        run: sed -ri 's#(FROM webispy/checkpatch@).*#\1${{ steps.build-and-push.outputs.digest }}#' Dockerfile
+      - name: Create Pull Request with updated Dockerfile
+        uses: peter-evans/create-pull-request@v4
+        with:
+          commit-message: Roll Dockerfile digest
```

Comment:
Thanks for letting me know about the `peter-evans/create-pull-request` action.

Please change the `commit-message` value to use following template because empty commit body is not allowed by `checkpatch.pl` tool.

    commit-message: |
        title

        body

And, please add some fields 

- `title:` - default title value of PR is always "Changes by create-pull-request action".
- `signoff: true` option to pass `checkpatch.pl` tool.

My recommendation(just an example) is below:

    title: Roll Dockerfile digest
    body: Apply ${{ steps.build-and-push.outputs.digest }}
    commit-message: |
        Roll Dockerfile digest

        ${{ steps.build-and-push.outputs.digest }}
        Apply docker image to action. 
    signoff: true
    base: ${{ github.event.repository.default_branch }}

Reply:
Thanks for looking at my PR! I update the PR with your suggestions!

Reply:
Thank you for update! But, the `base:` field that existed in the previous commit seems to be gone.
Is it intended?

Reply:
Oh, my bad, added it back now. I blame late night coding